README.md

```diff
@@ -65,6 +65,16 @@ All tests support the same set of arguments :
   * `-c,--check <0/1>` check correctness of results. This can be quite slow on large numbers of GPUs. Default : 1.
   * `-z,--blocking <0/1>` Make NCCL collective blocking, i.e. have CPUs wait and sync after each collective. Default : 0.
   * `-G,--cudagraph <num graph launches>` Capture iterations as a CUDA graph and then replay specified number of times. Default : 0.
+* Saving results
+  * `-F,--results_file` Define path for CSV file to persist nccl-tests result along with CLI arguments. Directory has to exist! Default: "" (Do not save to CSV file).
```

Comment:
I think this would be simpler:

`-F,--output_file <output file>`
Export results to CSV file. Default: "" (Do not save to CSV file).

Reply:
I think that would require to **not** print the normal output. So it's definitely a lot more work to support that.